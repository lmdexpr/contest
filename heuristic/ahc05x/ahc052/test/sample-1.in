30 10 10
13 25
7 14
17 22
0 18
29 1
3 25
14 22
14 29
26 2
3 10
00000000010000000000001000000
00000000010000000000001000000
00000000010000000000001000000
00000000010000000000001000000
00000000010000000000001000000
00000000010000000000001000000
00000000010000000000001000000
00000000000000000000001000000
00000000000000000000001000000
00000000000000000000001000000
00000000000000000000001000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000000000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
00000000000000010000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
011111111111111111100000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
111111111110000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
000000000000000000000000000000
