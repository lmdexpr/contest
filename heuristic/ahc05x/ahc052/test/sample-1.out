L L L R U R S D L S
R L L L D D L R D D
D S R D D S U D U R
L U S R D L S U L R
L U U L L U D U D L
L S U U S D L D U U
D D D D U D S D U S
U S U D S U L S D L
D R R R L L L S L S
L R S U S R D D R L
6
1
0
2
3
3
0
0
2
0
7
7
0
7
4
7
8
2
2
0
